source/client/process_impl.cc

```diff
@@ -902,6 +902,7 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const UriPtr& tracing_
     }
     ENVOY_LOG(debug, "Computed configuration: {}", absl::StrCat(bootstrap_));
     cluster_manager_ = cluster_manager_factory_->clusterManagerFromProto(bootstrap_);
+    THROW_IF_NOT_OK(cluster_manager_->initialize(bootstrap_));
```

Comment:
Let's not throw here. The pattern in this function is to handle errors in the following way:

```cc
if (!status.ok()) {
      ENVOY_LOG(error, "error message");
      return false;
}
```

Reply:
Done